### ISSUE TITLE
Change LK3 to a non-modifier key

### ISSUE THREAD

**Issue by sunjay** [clockworkpi/Keypad#2]
LK3 is bound to the Shift key. Since Shift is a modifier, it is really hard to detect when only LK3 is pressed. That means that it can't be used as an additional key, only as a modifier. There is already a shift key on the front of the device, so maybe it could be possible to configure it so that LK3 was usable as a regular key instead of a modifier?

**Comment by sfjuocekr** [comment]
I can change this very easily by adding the respective entries for LK3 and moving pin 5 from shift keys to keys.

When I wake up I'll modify the code and do a pull request!

**Comment by aboutafter** [comment]
What about the binding of the rest of the lightkeys? 

LK1, LK2, LK4 and LK5 are the same as the shift+ABXY keys.

**Comment by sfjuocekr** [comment]
You can actually bind them to any key you want, I have an updated sketch that rebinds pin 5 but have not tested it yet!

It indeed makes no sense to leave shift+ABXY mapped to LK if you have the extra keys... but not everyone has those ;)

**Comment by xelemorf** [comment]
I would really love to have separate mappings for the lightkey buttons, so by that, the shifted keys would be independent from the lightkey buttons. Keep up the good work! Are you doing this in Arduino IDE?

**Comment by aboutafter** [comment]
> You can actually bind them to any key you want, I have an updated sketch that rebinds pin 5 but have not tested it yet!
> 
> It indeed makes no sense to leave shift+ABXY mapped to LK if you have the extra keys... but not everyone has those ;)

The lightkeys should be 5 new keys (plus another 5 pressing the shift key). So we have potentially 10 new keys to bind which would be really usefull on RetroArch. Instead we only have 4 new keys (and you need to use them with Shift). 

By the way. Could this be updated on a new ClockworkOS version? or is it obligatory to update the keypad board? (because I don't know how to do that :( )

**Comment by xelemorf** [comment]
I fully agree, the lightkeys should be 5 new keys, but currently they are the same as A B X Y with shift modifier, you can check yourself over SSH using "sudo showkeys". 
The best approach would be to keep the shift modifier keys for A B X Y (and of course the normal A B X Y), but in addition, having 5 completely independent keys with the lightkey. Since board is behaving as a keyboard, this seems to be feasible.

On the other hand, since the keypad is a separate device and I think the gameshell only sends keycodes to the keypad (Arduino board), I think the keypad needs to be updated separately, and cannot be upgraded by the device OS itself (notice the keypad has a separate micro USB connector labelled "Arduino heart").

**Comment by sfjuocekr** [comment]
Changing the LK1-5 to represent five different keys is not hard to accomplish, simply change the KEYPAD_LIGHTx defines to what you want!

**Comment by xelemorf** [comment]
Thanks guys, I was able to modify the key mappings by your comments/guidance, it was super easy! :)
